api/src/main/kotlin/com/openlattice/organizations/JdbcConnectionParameters.kt

```diff
@@ -0,0 +1,56 @@
+package com.openlattice.organizations
+
+import com.fasterxml.jackson.annotation.JsonFilter
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.openlattice.authorization.securable.AbstractSecurableObject
+import com.openlattice.authorization.securable.SecurableObjectType
+import com.openlattice.client.serialization.SerializationConstants
+import java.io.Serializable
+import java.util.*
+
+/**
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+
+@JsonFilter("")
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+class JdbcConnectionParameters
+/**
+ * Creates an instance of a jdbc connection. All class properties must be serializable or marked as transient.
+ *
+ * @param _title Title of the connection.
+ * @param url Url of the server hosting the database.
+ * @param driver Name of the driver used to connect.
+ * @param database Name of the database to connect to.
+ * @param username Username used to log onto the database.
+ * @param password Password used to log onto the database.
+ * @param properties Object containing properties of the connection.
+ * @param description Description of the database.
+ *
+ */
+@JvmOverloads
+constructor(
+    @JsonProperty(SerializationConstants.ID_FIELD) val _id: UUID = UUID.randomUUID(),
+    @JsonProperty(SerializationConstants.TITLE_FIELD) val _title: String,
```

Comment:
Why the `_`?

Reply:
Without `_` JdbcConnectionParameters fails to compile because of an `Accidental override` error. 

I believe this happens because the Kotlin compiler tries to generate a getter for `val id` declared in the class primary constructor but the base class, `AbstractSecurableObject`, already has one. 

Not sure if there is a better way to avoid this, but I saw it was done this way in the Collaborations class so I followed it as convention 
https://github.com/openlattice/openlattice/blob/f89b8d18315fe3186a378ee10b26a80ce2319cf8/api/src/main/kotlin/com/openlattice/collaborations/Collaboration.kt#L12

One error output I saved for reference:
`Accidental override: The following declarations have the same JVM signature (getId()Ljava/util/UUID;`